src/detection.h

```diff
@@ -10,14 +11,32 @@
 
 #include "setup.h"
 
-/* Sensors */
-#define SENSOR_SIDE_LEFT 0
-#define SENSOR_SIDE_RIGHT 1
-#define SENSOR_FRONT_LEFT 2
-#define SENSOR_FRONT_RIGHT 3
+/* Sensors IDs*/
+#define SENSOR_SIDE_LEFT_ID 0
+#define SENSOR_SIDE_RIGHT_ID 1
+#define SENSOR_FRONT_LEFT_ID 2
+#define SENSOR_FRONT_RIGHT_ID 3
 #define NUM_SENSOR 4
 
+/* Calibration constants for sensors*/
+#define SENSOR_SIDE_LEFT_A 3.084
+#define SENSOR_SIDE_LEFT_B 0.375
+#define SENSOR_SIDE_RIGHT_A 2.870
+#define SENSOR_SIDE_RIGHT_B 0.361
+#define SENSOR_FRONT_LEFT_A 2.411
+#define SENSOR_FRONT_LEFT_B 0.278
+#define SENSOR_FRONT_RIGHT_A 2.462
+#define SENSOR_FRONT_RIGHT_B 0.293
```

Comment:
Maybe we should try to avoid `#define `s in header files unless they are really used somewhere else apart from within the associated `.c` file? (i.e.: I would move these defines to the `.c`)

Reply:
You are right, I moved these definitions to .c file.